.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,15 @@
+## Description of changes:
+
+## CTSM Issues Fixed (include github issue #):
+(https://github.com/ESCOMP/ctsm/issues)
+
+## Are answers expected to change?:
+
+## Any User Interface Changes?:
+
+## Externals updated (if any):
+
+## Testing performed:
+
+## Did you follow the conventions in the CTSM Developers Guide?:
```

Comment:
I think this last question is unnecessary. We should point to the developer's guide in the contributing document, but forcing everyone to type "Yes" here for every PR just seems kind of bothersome.